assets/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "amethyst_assets"
+version = "0.1.0"
+authors = ["torkleyy <torkleyy@gmail.com>"]
+readme = "README.md"
+description = """
+Asynchronous asset management for games.
+"""
+
```

Comment:
To keep `amethyst_assets` consistent with other crates, please don't forget to add:

```toml
license = "MIT/Apache-2.0"
keywords = ["game", "asset", "resource", "management", "amethyst"]
categories = ["filesystem"]

documentation = "https://www.amethyst.rs/doc/master/amethyst"
homepage = "https://www.amethyst.rs/"
repository = "https://github.com/amethyst/amethyst"

[badges]
appveyor = { repository = "amethyst/amethyst", branch = "develop" }
travis-ci = { repository = "amethyst/amethyst" }
```